js/scripts/generate-emoji-map.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs');
+const path = require('path');
+const twemoji = require('twemoji/2/twemoji.npm');
+
+const outputPath = './src/forum/generated/emojiMap.json';
+const data = require('emojibase-data/en/data.json');
+const twemojiFileNames = fs.readdirSync('./node_modules/twemoji/2/svg')
+  .map(name => path.basename(name, '.svg'));
+
+const alternative = {
+  "👁️‍🗨️" : "👁‍🗨",
+};
+
+const emojis = {};
+
+for (let e of data) {
+  const {
+    name,
+    shortcodes,
+    tags,
+  } = e;
```

Comment:
This can be simplified to const { shortcodes } = e

Reply:
and I wish for eslint